Samples/Blinky/Blinky-samc21xpro.lpi

```diff
@@ -73,33 +73,31 @@
   <CompilerOptions>
     <Version Value="11"/>
     <Target>
-      <Filename Value="Blinky"/>
+      <Filename Value="bin/$(TargetCPU)-$(TargetOS)/samc21/$NameOnly($(ProjFile))"/>
     </Target>
     <SearchPaths>
       <IncludeFiles Value="../../Source;$(ProjOutDir)"/>
       <OtherUnitFiles Value="../../Source"/>
-      <UnitOutputDirectory Value="lib/$(TargetCPU)-$(TargetOS)"/>
+      <UnitOutputDirectory Value="lib/$(TargetCPU)-$(TargetOS)/samc21"/>
     </SearchPaths>
     <Parsing>
       <SyntaxOptions>
         <UseAnsiStrings Value="False"/>
       </SyntaxOptions>
     </Parsing>
     <CodeGeneration>
-      <HeapSize Value="8192"/>
```

Comment:
Is there a reason why you removed Heap/StackSize Values?

Reply:
Not anything particular.
My normal practice is to try to get things running with as little extra settings as possible.
So, its just a way for me to check if things are running without these extra settings.

Reply:
I am too lazy to look this up in details atm but I guess your code will sooner or later crash and burn when those are not properly set. But it may be that some defaults are applied when nothing is set, as said, too lazy to look this up atm.